### ISSUE TITLE
Text encoder not defined when creating a file

### ISSUE THREAD

**Issue by Cooper-Kunz** [hashgraph/hedera-sdk-js#139]
I'm porting examples from the Java SDK for release `v1.0.0-beta.2`, as part of #134 

When attempting to run a `FileCreateTransaction()` I get the following error. 

```
(node:1356) UnhandledPromiseRejectionWarning: ReferenceError: TextEncoder is not defined
    at FileCreateTransaction.setContents (/Users/cooperkunz/Desktop/Hedera-JS-Examples/js-sdk/node_modules/@hashgraph/sdk/lib/file/FileCreateTransaction.js:47:33)
    at main (/Users/cooperkunz/Desktop/Hedera-JS-Examples/js-sdk/file-create.js:22:10)
    at Object.<anonymous> (/Users/cooperkunz/Desktop/Hedera-JS-Examples/js-sdk/file-create.js:28:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
    at startup (internal/bootstrap/node.js:283:19)
```

Full code, similarly referenced in #125 

```
require("dotenv").config();

const { Client, FileCreateTransaction } = require("@hashgraph/sdk");

async function main() {
    const operatorPrivateKey = process.env.OPERATOR_KEY;
    const operatorAccount = process.env.OPERATOR_ID;

    if (operatorPrivateKey == null || operatorAccount == null) {
        throw new Error("environment variables OPERATOR_KEY and OPERATOR_ID must be present");
    }

    const client = new Client({
        network: { "0.testnet.hedera.com:50211": "0.0.3" },
        operator: {
            account: operatorAccount,
            privateKey: operatorPrivateKey
        }
    });

    const transactionId = await new FileCreateTransaction()
        .setContents("test file create")
        .execute(client);

    console.log(`transactionId = ${transactionId}`);
}

main();
```

Note: the error is also encountered on release `v1.0.0-beta.1`

**Comment by mehcode** [comment]
@Cooper-Kunz Thanks for the report. This was fixed on master late yesterday and I released that as `v1.0.0-beta.3` just now. 

This was the result of us using a newer API that is only supported on newer versions of Node. CI is now run against LTS node and master/v1.0.0-beta.3 pass.

**Comment by Cooper-Kunz** [comment]
Hey @mehcode - thanks for the quick response.

Attempting to run the same code above on `v1.0.0-beta.3` results in an `INSUFFICIENT_TX_FEE (9)`

I would expect the default tx fee to be sufficiently high for most cases, e.g. the 16 char string.
Can we fix this as well? It should return the excess hbar regardless.

Additionally - could you clarify the difference between `Hbar.to(100)` and `Hbar.from(100)`?

---

I attempted to run the following code, updating the max tx fee to 1000ℏ

```
require("dotenv").config();

const { Client, FileCreateTransaction, Hbar } = require("@hashgraph/sdk");

async function main() {
    const operatorPrivateKey = process.env.OPERATOR_KEY;
    const operatorAccount = process.env.OPERATOR_ID;

    if (operatorPrivateKey == null || operatorAccount == null) {
        throw new Error("environment variables OPERATOR_KEY and OPERATOR_ID must be present");
    }

    const client = new Client({
        network: { "0.testnet.hedera.com:50211": "0.0.3" },
        operator: {
            account: operatorAccount,
            privateKey: operatorPrivateKey
        }
    });



    const transactionId = await new FileCreateTransaction()
        .setContents("test file create")
        .setMaxTransactionFee(Hbar.to(1000))
        .execute(client);

    console.log(`transactionId = ${transactionId}`);
}

main();
```

Which results in a successful transaction -https://testnet.dragonglass.me/hedera/transactions/0012491577478540623000000
